sos/report/plugins/ceph_mon.py

```diff
@@ -179,6 +179,10 @@ def setup(self):
             [f"ceph tell mon.{mid} mon_status" for mid in self.get_ceph_ids()],
             subdir="json_output",
         )
+        self.add_cmd_output(
+            [f"ceph tell mon.{mid} sessions" for mid in self.get_ceph_ids()],
+            subdir="json_output",
+        )
```

Comment:
Since we're iterating over the same set of ids here, we should not make the call twice if possible, e.g.

```python
for mid in self.get_ceph_ids():
    self.add_cmd_output([
        f"ceph tell mon.{mid} mon_status",
        f"ceph tell mon.{mid} sessions",
    ], subdir="json_output")
```

Or, perhaps

```python
subcmds = [
    'mon_status',
    'sessions',
]
for mid in self.get_ceph_ids():
    self.add_cmd_output(
        [f"ceph tell mon.{mid} {subcmd}" for subcmd in subcmds],
        subdir='json_output'
    )
```